### PR TITLE
Automations - enable all SCT2 labels in all SMR displays

### PR DESCRIPTION
**Pull request by 19wintersp**
# Summary of changes

Adds a script to the AIRAC workflow which will automatically add all labels in the SCT file to all vSMR display ASRs.

The script also avoids unnecessarily duplicating identical labels within the ASR, which EuroScope includes when saving but appears to ignore when loading. I've checked this behaviour in my EuroScope and it appears to work normally: the ASR loads without issue, and the "SCT2" item shows up with a black tick in the "Display settings dialog" dialog.

Also cleaned up other workflows, comments welcome.

### CODE REVIEW

**Comment by luke11brown** [comment]
I like.

Only issue I have is that we may in the future add more sector files to the folder structure (probably in the same folder for simplicity) to cover LXGB for example. How will the script handle two sector files in the same folder?

i.e. It would need to add LXGB labels to LXGB SMR profiles, but exclude the labels pulled from UK.sct.

**Comment by 19wintersp** [comment]
It should be possible to consider separate sector files together, and so include the "union" of all sets of labels in every ASR, since ES appears to ignore labels which don't exist. If that doesn't work, I'd instead suggest including a comment (or unrecognised property) in each ASR specifying which sector file is to be used, or having the script select ASRs via their respective PRFs, so that the relevant sector file for each can be taken from the PRF. That is to say, it should be implementable without much issue, and the preferred solution is probably best considered when the problem needs solving 😄

One thing I forgot to add is that I haven't checked properly that the updated workflows work, so extra scrutiny there may be necessary.

**Comment by luke11brown** [comment]
Further thought. We currently have all the SCT2 data defined in the ASR files.

Is it wise to remove this initially and then get the script to generate a clean sheet?

**Comment by 19wintersp** [comment]
The script expects the SCT2 labels from the previous cycle to be present to allow the new labels to be positioned correctly within the file (line 26), and will remove all existing SCT2-related lines before adding the new ones (line 25). This is in line with the current convention of the main branch reflecting the contents of the Pack as distributed in the most recent cycle, including all automatic output. That is, the script should be run on the repo as-is, as this is the same context in which it will be run in future cycles (ie. its own output, or equivalent).

**Comment by luke11brown** [comment]
> Something I've just realised is that there are some SMR displays—mostly (only?) military—which do not use vSMR, and so are ignored by the script. Is this something that should be changed?

I'll be honest I neglect the Mil stuff as it's something I'm not particularly interested in. @Robbo599 - thoughts? We can make the Mil stuff use vSMR for simplicity using another PR and merge before this.

**Comment by 19wintersp** [comment]
Changing the Mil displays to use vSMR could be done before or after this PR as necessary, without affecting anything; the script will simply ignore non-vSMR ASRs.

File: .github/workflows/airac.yml [hunk 1]
```diff
@@ -1,40 +1,52 @@
...
+          python workflows/airac/auto_airac_actions.py
           echo "airac=$(cat airac.txt)" >> "$GITHUB_ENV"
           rm airac.txt
 
       - name: Commit changes
         run: |
-          git config --local user.email "privileged-access@vatsim.uk"
-          git config --local user.name "VATSIM UK"
           git add .
-          git commit -m "Sector file update for AIRAC ${{ env.airac }}"
```
Comment (luke11brown) [inline .github/workflows/airac.yml:37]: Reason for deleting these lines?

**Comment by luke11brown** [comment]
Could we also expand the script to enable the fies/labels on displays? I believe @KHardern had a script to do this previously.

i.e.
all *2.asr and *3.asr should have fixes included
all *4.asr should have fixes and fix labels included

exceptions are listed in .data/fixes_exclude.txt

File: .github/workflows/airac.yml (hunk 1 above)
Comment (19wintersp) [reply .github/workflows/airac.yml:37]: The `git config` lines have been moved further up the file, as they are common with both commits. The `git commit` line is on the line below, with an updated message.

**Comment by 19wintersp** [comment]
That would be better done as a part of a new script in the workflow. As far as I know, doing so would require the additional information present in the sector file repository (and so require cloning that repository in the workflow), where the fixes are placed in separate files based on their classification. If that's acceptable, I'm happy to add it.

I would note that this is starting to move towards generating more and more of the ASRs; whilst I think this is a good long-term project for the Pack, it's somewhat outside of the scope of this particular change.

**Comment by luke11brown** [comment]
ok, we'll give it a whirl this Thursday

**Pull request merged**